mlir/tools/xmir-runner/xmir-runner.cpp

```diff
@@ -0,0 +1,102 @@
+//===- xmir-runner.cpp - MLIR ROCM Execution Driver-------------------===//
```

Comment:
Now wait just a darn minute, why is this a separate executable instead of being part of `mlir-rocm-runner`?